x-pack/qa/core-rest-tests-with-security/src/yamlRestTest/java/org/elasticsearch/xpack/security/CoreWithSecurityClientYamlTestSuiteIT.java

```diff
@@ -47,8 +56,10 @@ public class CoreWithSecurityClientYamlTestSuiteIT extends ESClientYamlSuiteTest
         .setting("xpack.license.self_generated.type", "trial")
         .setting("xpack.security.autoconfiguration.enabled", "false")
         .user(USER, PASS)
+        .user("get_alias_test_user", "x-pack-test-password", "get_alias_test_role", false)
```

Comment:
The users and roles must be defined in the test suit in order to avoid causing failures for tests which do not run with the security enabled.